Formula/akka.rb

```diff
@@ -21,16 +21,16 @@ class Akka < Formula
     if OS.mac?
         if Hardware::CPU.intel?
             url "https://downloads.akka.io/#{version}/akka_darwin_amd64_#{version}.tar.gz"
-            sha256 "cd96fc19a937cf0c54c10723ab7462a5d857653df868a787ee570812b184367b"
+            sha256 "4efd6996e104197dfb6e1e48c6c4b6e2c7537de70229298e804098fe54de9635"
         else
             url "https://downloads.akka.io/#{version}/akka_darwin_arm64_#{version}.tar.gz"
-            sha256 "9e8507fb3fa44f064b6266f5c9f3e66c9f93bfdb9e19c84a59dd3b7e26b43b62"
+            sha256 "94b36a5c27bf2ac7c9f5176b02c073bb47a70d6b10058002dfd4747e611a638d"
         end
     end
 
     if OS.linux?
         url "https://downloads.akka.io/#{version}/akka_linux_amd64_#{version}.tar.gz"
-        sha256 "b96ef9c7b722b9d828fede3d326edb35d967db345c585d116b23ead7b0bf4816"
+        sha256 "0f72cf0d2c0d7dd033956b7138411d8b4b8e157a6ef19866f9abb061f76c74c1"
```

Comment:
took the shas from https://ff1726b70c5b20167680c28535ee42a918a8f6aae2e95a08dd3dc1c-apidata.googleusercontent.com/download/storage/v1/b/kalix-binaries/o/3.0.4%2Fakka_checksums.txt?jk=AXvcXDtjcV4ERK8jU-wkro-pryEbQ64cQW1RLT3IAHolGM1Guad8TSAUjYSQeDBoj8T074CAciegbUebR2RxF1VF5O86B9NdDNu3mAcM9WmyumXB7iZC9xzTw3jEevTHy48mmOBO9MvBRr6fV7HsjQpNQ5MhDdILCRT_md1JMAR5kyened7r9t5wbaJ7CvMbKZoxqriFZ-H3iNOCe8C4btYTtrbky7SkbXQX-9s1KZ8C4lV-J272909frELWW0oNf6KpGRGPm1OZ9-Edzw-RD-gXw3aVsJ9rI5MIEKv7g2gQz0oNBo97okDbg2rYKxj_hKbUI8MBeRclF_-62fn74qLWqnD5yHjHQN_O7s4NuHWoG1-ESa1e11m5LHaoondk4weWlievVCQ9Y1ghdy7D6ScXvtABucx7QuXNDeXj4yC_Q07QD9cyoiJPdjiH_bZgLi5VVhtzZvqzLq3XDdJNYXFl3LcL99ycaSx11hnURcK1dDhSxxoKUb6fvsFoNtUVohWrr-JeiaBIfvwAstLyFAHYWV4jlUU53-SOD5_UApiKZsv11ceyutC6YirIv3Pc8z2Ladij5fNsxnvFf6ZM3LNyWQOfQ_sdKcQuNDlHH2bO_HdFeuDJS55AyKA3OQ1AYBr1f7kZ8u9FN8HuQre8KKChRJ-WLwrncHnEru6VEV7Cy_tI8G44R6q5sWteiZVpiGX15A8Z5FdY19g4pKH-XVUKhyOBGZHfvUC9nGsOvgKjaDcBQ_zWrYklC8kifXBsCcV7IYRqjIHwSUk_wvoP1zTM9fwc62KdsNrfxzPorCEIcvaPjoHhVc_l26nV7BmkPbgGeLQlqlOyYsfROT4l4rPJsiQXL3XIxp7o2L-Oeh0z6OGVgQj5H_HrHAO_5ZKk3OCpNts52BOXmM9TyshXKe3wKNOjq5xt9MFdhNyiTe__8pkLrtqM8UKc7LqbYcrpWDPjJ72ZPMJikumUHE9-RxX-SlYPeoMPzb0kJxe6nSye7VSg3EvlfOhk9-DB3rXDud3TV7F3HScHuqJRzc9LT5mkvNFp0y7eB1bYDjvfOsOXGAN-JfRO7gUHWFD7l9DMkoU8JpPYQLuHlYYvuTVJx3zYpJJxte5PKiCgtdPjTmqSuXDRBdqPmoDoL2t427KrwrP-EHgvchTQMlLUK4uU1E-0vHbyB5vuKxZTrg&isca=1